src/map/merit.cpp

```diff
@@ -195,9 +198,9 @@ void CMeritPoints::LoadMeritPoints(uint32 charid)
 
     for (uint16 i = 0; i < MERITS_COUNT; ++i)
     {
-        if ((catNumber < 51 && i == meritNameSpace::groupOffset[catNumber]) || (catNumber > 25 && catNumber < 31))
+        if ((catNumber < 54 && i == meritNameSpace::groupOffset[catNumber]) || (catNumber > 26 && catNumber < 31) || catNumber == 51)
         {
-            if (catNumber > 25 && catNumber < 31)
+            if (catNumber > 26 && catNumber < 31 || catNumber == 51)
```

Comment:
I don't know anything about these systems, what's with this seemingly random range?

Reply:
Its because there is a few gaps in the merit category range so that part points to the correct category, in this case cat 1 & cat 2 merits and the rest are diferent types, like HP/MP and other stuff, its been a while since anyone messed with these tbh, probly could do with a look at later on down the line.

Reply:
Might be worth getting some of the more versed guys to explain it, as this was all way before my time

Reply:
Tummie would know more as he was the one that had these numbers when they got added to retail